geoh5py/__init__.py

```diff
@@ -17,15 +17,32 @@
 
 # flake8: noqa
 
-__version__ = "0.10.0-alpha.1"
-
 import inspect
 
 from geoh5py.workspace import Workspace
 
 from . import groups, objects
 from .groups import CustomGroup
 
+# Define the variable '__version__' at the runtime
+try:
+    from setuptools_scm import get_version
```

Comment:
**thought:**
I understand from the "Hatch VCS Footgun Example" that this whole block is it to provide an accurate version from the git tag in editable mode when moving between revisions.

However, will the attempt of setuptools_scm to find a git tag slow down the first import of geoh5?
And if applying the same mechanism across all our package, how much will it slow down the first final application which imports them all?

Trying to have the proper version in edit mode for developers might end up by as a slowdown for the end users. I am no convinced it is worth.

Another preferable approach could be a [post_checkout](https://git-scm.com/docs/githooks#_post_checkout) git hook, that would execute `setuptools_scm.get_version` to patch a possibly existing `_version` file.

The editable use case does not need to be solved now.